src/utils/util.js

```diff
@@ -32,6 +32,15 @@ export function isObject (value) {
     return value !== null && (type === 'object' || type === 'function');
 }
 
+export function getFloat32ArrayFromArray (array) {
+    const length = array.length;
```

Comment:
Can we just? 

```js
getFloat32ArrayFromArray (array) {
  return new Float32Array(array);
}
```

Reply:
or in any case use [from](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/from) 

```js
getFloat32ArrayFromArray (array) {
  return new Float32Array.from(array)
```

Reply:
The first one, of course not, you are doing the same and you are hitting the same monkey patch, getting the same results.

Reply:
The second one though makes a lot of sense, and I actually tried it before, but no luck, the performance is bad too, core-js is probably monkey patching that too.

Reply:
👍 good to know :)